### ISSUE TITLE
Crash when creating a search folder

### ISSUE THREAD

**Issue by sparusaurata** [lwindolf/liferea#431]
Each time I try to create a search folder, Liferea crashes immediately.
When launching it the terminal, I get :

```
** (liferea:5498): ERROR **: Loading /usr/share/liferea/search_folder.ui failed: L'ouverture du fichier « /usr/share/liferea/search_folder.ui » a échoué : Aucun fichier ou dossier de ce type
[1]    5498 trace trap  liferea
```

I use Linux Mint 18 with Cinnamon 3.0.7.

**Comment by lwindolf** [comment]
We recently split up UI files in git master. So master (1.12-RC*) is different from 1.11 regarding .ui files.

If you mix up both versions e.g. by trying 1.12 from source and not fully "make install" them above effect can happen.

**Comment by Leiaz** [comment]
This was fixed by commit b22c827.
